README.md

```diff
@@ -15,25 +15,43 @@ These benchmarks can be executed on a vanilla Kubernetes or OpenShift cluster. I
   * Non-OpenShift Deployments Only; Optional: [Cadvisor](https://github.com/google/cadvisor)
 
 * Notes
-   * Clone git repositories into sibling directories to the `loki-benchmarks` one.
-   * Recommended cluster size: `m4.16xlarge`
+  * Clone git repositories into sibling directories to the `loki-benchmarks` one.
+  * Recommended cluster size: `m4.16xlarge`
 
 ## Configuring Tests
 
 To change the testing configuration, see the files in the [config](./config) directory.
 
-Use the `scenarios/benchmarks.yaml` file to add, modify, or remove configurations. Modify the `generator.yaml`, `metrics.yaml`, or `querier.yaml` in the prefered deployment method directory to change these soruces.
+Different scenarios can be customized under `config/benchmarks/scenarios/benchmarks`. Current the benchmarks support two testing scenarios:
+
+* Ingestion path scenarios: [suppored configuration](https://github.com/observatorium/loki-benchmarks/blob/1a0a9e8f6190475b6c1bfacb5a31a88bd76cbb36/internal/config/config.go#L76-L81), this test will generate X amount of logs throughout a 30 minute window that's supposed to represent a full day of log ingestion.
+* Query path scenarios: [supported configuration](https://github.com/observatorium/loki-benchmarks/blob/1a0a9e8f6190475b6c1bfacb5a31a88bd76cbb36/internal/config/config.go#L102-L108), the theory behind this test is to generate the amount of data that would be queried before it starts running the queries.
 
 ## Running Benchmarks
 
-Use the `make run-rhobs-benchmarks` or `make run-operator-benchmarks` to execute the benchmark program with the RHOBS or operator deployment styles on OpenShift respectively. Upon successful completion, a JSON and XML file will be created in the `reports/date+time` directory with the results of the tests.
+### Prerequisites
 
-## Troubleshooting
+The `run-operator-benchmarks` expects the following two env vars to be set `LOKI_OPERATOR_REGISTRY` `LOKI_STORAGE_BUCKET`.
+E.g
+
+```shell
+export LOKI_OPERATOR_REGISTRY=jmarcal
```

Comment:
Suggest to rename this to `LOKI_OPERATOR_REGISTRY_ORG`

Reply:
Ended up going with `LOKI_OPERATOR_REGISTRY_BASE` since it's more similar to how we use `make olm-deploy`

Reply:
Sounds good for me